kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSAnnotationImpl.kt

```diff
@@ -27,6 +27,7 @@ class KSAnnotationImpl(private val annotationApplication: KtAnnotationApplicatio
     override val arguments: List<KSValueArgument> by lazy {
         annotationApplication.arguments.map { KSValueArgumentImpl(it) }
     }
+    override val defaultArguments: List<KSValueArgument> = emptyList()
```

Comment:
Similar question as the other PR. I don't know if this is correct because I don't have context on this module

Reply:
as mentioned in previous PR, you can put a TODO() here.

Reply:
Done